### PR TITLE
Add dialog before drafting email, if not all progress items are completed.

### PR DESCRIPTION
**Pull request by dlaliberte**
This adds a dialog to the chromedash-process-overview component based on the design in https://docs.google.com/document/d/1eh8uT_DiJncvUi5ZZicLBkX06nepD8Lza8XR40yI08c/edit?resourcekey=0-IpKO8GjpLX-Qg9_Is-UHUw

### CODE REVIEW

File: static/elements/chromedash-process-overview.js
Comment (jrobbins) [inline static/elements/chromedash-process-overview.js:265]: This button label is kind of long.
How about just: "Continue" or "Proceed"?

Also, this seems like a case where the button should have a border, but not be filled in to look like the primary button.    And, the buttons are crowded too close together.

Try:
margin: var(--content-padding);
border: var(--default-border);
border-color: var(--primary-border-background);
padding: 4px 16px;
border-radius: var(--border-radius);

You could just do that here in this component, or if you are feel like diving into our CSS you could add some "outlined button" variables to theme.css and then define a new button CSS class in shared.scss.

**Review by jrobbins — CHANGES_REQUESTED** [review]
Overall this looks good.   I have some wording and CSS comments.

I think the main thing blocking this from landing is that I need to go through the list of progress items and narrow them down and make sure that they are all actionable.  I'll try to get that PR sent tomorrow.

File: static/elements/chromedash-process-overview.js
Comment (dlaliberte) [reply static/elements/chromedash-process-overview.js]: done

**Comment by dlaliberte** [comment]
Here is a screenshot with the title and button style changes.

image

**Comment by jrobbins** [comment]
This all looks good.  I just need to revisit the progress detectors before we merge and deploy this.

**Comment by dlaliberte** [comment]
I'm thinking the dialog is lacking a bit of useful context.  Maybe we should add a sentence at the top, something like:  "Before proceeding to Draft Intent to Experiment email, note that there are one or more fields missing:".  That way, the reason for the dialog, and the meaning of the "Edit fields" and "Proceed" buttons becomes clear.

I also see there is a bit of extra whitespace below the buttons.  I'll check that out.

**Comment by dlaliberte** [comment]
Another thing to note is that the "One LGTM on Intent to Experiment" won't be possible without having first sent out the email, so this field will always be missing.     Should we skip all items after the "...email" item?

**Comment by jrobbins** [comment]
We could omit everything after the "... email" progress item.  Or, we could add a new boolean to each progress item named something like is_intent_prerequisite.

**Comment by jpmedley** [comment]
Sorry I didn't find this earlier. What does this do with milestone fields?

**Pull request closed without merging**
